va/va_backend.h

```diff
@@ -631,7 +631,14 @@ struct VADriverContext
     void (*info_callback)(VADriverContextP pDriverContext,
                           const char *message);
 
-    unsigned long reserved[38];         /* reserve for future add-ins, decrease the subscript accordingly */
+    /**
+     * \brief The VA/Protected implementation hooks.
+     *
+     * This structure is allocated from libva with calloc().
+     */
+    struct VADriverVTableProt *vtable_prot;
```

Comment:
pointer is 8bytes in 64bit build, need adjust reserved size according to 32/64bit build. carl should have reference usage

Reply:
This is follow libva convention. reserved is defined as unsigned long which size is same as pointer size.
You can reference [this change](https://github.com/intel/libva/commit/afca400d5629409ae7d51c4b1109a1b73ab2844a#diff-2cb795fdf59e7e486aef9880f1a0fbe0d2a39a803fa6c5165aa16a968ad98010) or [this change](https://github.com/intel/libva/commit/6650b390d1c21ce4abc78234fbc74e02cf25ddca#diff-2cb795fdf59e7e486aef9880f1a0fbe0d2a39a803fa6c5165aa16a968ad98010)